tests/unit_tests/test_policy.py

```diff
@@ -21,11 +23,21 @@ def add_example_policy_paths():
     sys.path.append(os.path.join(POLICY_PATH, 'example_policy'))
 
 
-def _load_task_and_apply_policy(config_path) -> sky.Dag:
+def _load_task_and_apply_policy(
+        config_path: str,
+        idle_minutes_to_autostop: Optional[int] = None) -> sky.Dag:
     os.environ['SKYPILOT_CONFIG'] = config_path
     importlib.reload(skypilot_config)
     task = sky.Task.from_yaml(os.path.join(POLICY_PATH, 'task.yaml'))
-    return dag_utils.convert_entrypoint_to_dag_and_apply_policy(task)
+    return policy_utils.apply(
+        task,
+        operation_args=sky.OperationArgs(
```

Comment:
Are users/admins expected to use this class? If not, how about not exposing it at root level?

Reply:
An admin should use an object of this class in a policy, but does not need to actually use this class. I was thinking to expose it for type hints, but may not be necessasry.